Neo4j.Driver/Neo4j.Driver.Tests/ConnectionPoolTests.cs

```diff
@@ -665,8 +665,8 @@ public class NumberOfInUseConnections
             public void ShouldReturnZeroAfterCreation()
             {
                 var uri = new Uri("localhost:7687");
-                var connectionSettings = new ConnectionSettings(uri, AuthTokens.None, Config.DefaultConfig);
-                var poolSettings = new ConnectionPoolSettings(1, 1, TimeSpan.MaxValue, TimeSpan.MaxValue);
+                var connectionSettings = new ConnectionSettings(AuthTokens.None, Config.DefaultConfig);
+                var poolSettings = new ConnectionPoolSettings(1, 1, Config.Infinite, Config.Infinite, Config.Infinite);
```

Comment:
The current decision is that we treat all negative values as Infinite, which is stated in the API docs of `Config` for these `TimeSpan` settings.
The `Config.Infinite` value is `internal` and as you figured out I get this number from some standard lib.